Makefile

```diff
@@ -160,50 +160,37 @@ src :=  corev_apu/tb/axi_adapter.sv
-        vendor/pulp-platform/tech_cells_generic/src/deprecated/cluster_clk_cells.sv         \
-        vendor/pulp-platform/tech_cells_generic/src/deprecated/pulp_clk_cells.sv            \
-        vendor/pulp-platform/tech_cells_generic/src/rtl/tc_clk.sv                           \
+        vendor/pulp-platform/axi/src/axi_multicut.sv                                 \
+        vendor/pulp-platform/common_cells/src/rstgen_bypass.sv                       \
+        vendor/pulp-platform/common_cells/src/rstgen.sv                              \
+        vendor/pulp-platform/common_cells/src/addr_decode.sv                         \
+        vendor/pulp-platform/axi/src/axi_cut.sv                                      \
+        vendor/pulp-platform/axi/src/axi_join.sv                                     \
+        vendor/pulp-platform/axi/src/axi_delayer.sv                                  \
+        vendor/pulp-platform/axi/src/axi_to_axi_lite.sv                              \
+        vendor/pulp-platform/axi/src/axi_id_prepend.sv                               \
+        vendor/pulp-platform/axi/src/axi_atop_filter.sv                              \
+        vendor/pulp-platform/axi/src/axi_err_slv.sv                                  \
+        vendor/pulp-platform/axi/src/axi_mux.sv                                      \
+        vendor/pulp-platform/axi/src/axi_demux.sv                                    \
+        vendor/pulp-platform/axi/src/axi_xbar.sv                                     \
+        vendor/pulp-platform/common_cells/src/cdc_2phase.sv                          \
+        vendor/pulp-platform/common_cells/src/spill_register_flushable.sv            \
+        vendor/pulp-platform/common_cells/src/spill_register.sv                      \
+        vendor/pulp-platform/common_cells/src/deprecated/fifo_v1.sv                  \
+        vendor/pulp-platform/common_cells/src/deprecated/fifo_v2.sv                  \
+        vendor/pulp-platform/common_cells/src/stream_delay.sv                        \
+        vendor/pulp-platform/common_cells/src/lfsr_16bit.sv                          \
+        vendor/pulp-platform/tech_cells_generic/src/deprecated/cluster_clk_cells.sv  \
+        vendor/pulp-platform/tech_cells_generic/src/deprecated/pulp_clk_cells.sv     \
+        vendor/pulp-platform/tech_cells_generic/src/rtl/tc_clk.sv                    \
         corev_apu/tb/ariane_testharness.sv                                           \
         corev_apu/tb/ariane_peripherals.sv                                           \
         corev_apu/tb/rvfi_tracer.sv                                                  \
         corev_apu/tb/common/uart.sv                                                  \
         corev_apu/tb/common/SimDTM.sv                                                \
         corev_apu/tb/common/SimJTAG.sv
 
-# SV32 MMU for CV32, SV39 MMU for CV64
```

Comment:
Why are these removed? Intentional or accidental?

Reply:
Intentional, these were module duplication in the Makefile and CVA6 Flist